### ISSUE TITLE
Consider supporting SPARQL 1.1 Update

### ISSUE THREAD

**Issue by jindrichmynarz** [tarql/tarql#67]
A useful feature in tarql can be support of [SPARQL 1.1 Update](https://www.w3.org/TR/sparql11-update/). Currently, tarql is limited to (a series of) CONSTRUCT queries. Supporting SPARQL 1.1 Update would open many useful ways of processing data. An example use case that I'm considering is producing data in multiple named graphs, which is currently not possible with CONSTRUCT queries.

The output could be serialized to any standard serialization of RDF quads, possibly [N-Quads](https://www.w3.org/TR/n-quads) to allow streaming serialization.

If support of the complete SPARQL 1.1 Update would introduce too much complexity to tarql, supporting only (a series of) [INSERT](https://www.w3.org/TR/sparql11-update/#insert) operations would still deliver a lot of value.

**Comment by VladimirAlexiev** [comment]
Updating a graph is useful in pipelines that need to update data.
However, tarql cannot join CSV data to real data in a repo (#25), and N-Quads cannot carry DELETE statements.
So I think you mean only INSERT: or @jindrichmynarz do you have an example with graph update?

- Jena supports GRAPH in CONSTRUCT. So it may be easier to implement #98.
- As a workaround, one can save turtle, then can use the [SPARQL Graph Store Protocol](https://www.w3.org/TR/sparql11-http-rdf-update/) to overwrite the respective graph

**Comment by jindrichmynarz** [comment]
Yes, I meant only INSERT statements. CONSTRUCT GRAPH statements would work the same.
